contrib/libxmp/src/loaders/s3m_load.c

```diff
@@ -383,54 +383,16 @@ static int s3m_load(struct module_data *m, HIO_HANDLE * f, const int start)
 		m->quirk |= QUIRK_VSALL;
 	}
 
-#ifndef LIBXMP_CORE_PLAYER
 	switch (sfh.version >> 12) {
 	case 1:
-		snprintf(tracker_name, 40, "Scream Tracker %d.%02x",
```

Comment:
NOTE: Unlike the "core" code, my goal was to continue supporting the various quirks - just without accurate printing of the tracker name, which we never use.